### ISSUE TITLE
Update to latest scipy

### ISSUE THREAD

**Issue by ebolyen** [qiime2/q2-deblur#68]
**Bug Description**
Currently pinned to an old scipy:
```
    # scipy isn't directly used in this plugin, but setting a version pin here
    # because deblur doesn't currently work with modern scipy versions.
    - scipy <1.1.0
```

**Questions**
1. Why is this pin needed? Is it related to skbio?

**Comments**
1. This isn't super critical (yet!), but it'll matter more the older this issue gets.

**Comment by mortonjt** [comment]
Can we close this issue? There was a recent release of deblur in Sept 2 weeks after this issue was raised

https://github.com/biocore/deblur/releases

**Comment by thermokarst** [comment]
Fixed in #76
